IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/WebElementExtensions.cs

```diff
@@ -48,15 +48,44 @@ public static void ScrollIntoView(this IWebElement element, IWebDriver driver, i
             js.ExecuteScript($"window.scrollTo(0,{position})");
         }
 
+        /// <summary>
+        /// Clears the current text, 
+        /// uses SendKeys to send the specified value to the element, 
+        /// </summary>
+        /// <param name="value">Value to send to the element</param>
+        public static void ClearAndSendKeys(this IWebElement element, string value)
```

Comment:
Personally I hate seeing conjunctions in method names as it is often an indication that a method is doing too much. However in this case perhaps consider a different name such as "SendKeysReplace"?